package.json

```diff
@@ -176,19 +176,19 @@
     "jest-environment-jsdom": "^29.6.2",
     "jest-fetch-mock": "^3.0.3",
     "json-diff": "^1.0.6",
-    "lint-staged": "^13.2.2",
-    "msw": "^1.2.5",
+    "lint-staged": "^14.0.1",
+    "msw": "^1.3.0",
```

Comment:
if this causes a test to fail you will need to catch up with @mantagen , he was looking at this